src/pages/QuestionPage.tsx

```diff
@@ -51,14 +65,59 @@ function QuestionPage() {
 
     // wait 1 second before navigate
     setTimeout(() => {
-      setLoading(false)
-      setAnswer("")
-      navigate(`/${nextQuestionId}`)
+      setLoading(false);
+    setAnswer("");
+
+    // Check if the current question is the last one
+    if (questionId >= totalQuestion - 1) {
+      navigate('/thankyou');
+    } else {
+      navigate(`/${nextQuestionId}`);
+    }
     }, 1000)
   }
 
+  const goToPrevQuestion = () => {
+    setLoading(true)
+
+    setAnswers(
```

Comment:
this code block need to refactor

Reply:
i just copy paste on function nextQuestion mister